### PR TITLE
Add Fremont, CA template

### PR DESCRIPTION
**Pull request by serifluous**

### CODE REVIEW

File: _emails/fremont-ca.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  Sincerely,
+
+  \[YOUR NAME\]
+
+  \[YOUR ADDRESS\]
+
+  \[YOUR EMAIL\]
+
+  \[YOUR PHONE NUMBER\]
+layout: email
```
Comment (teddywilson) [inline _emails/fremont-ca.md:44]: Let's put this at the top with the rest of the attributes

File: _emails/fremont-ca.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  I join the calls of those across the country to #DefundThePolice. I demand a budget that adequately and effectively meets the needs of at-risk Fremont residents during this trying and uncertain time, when livelihoods are on the line. I call on you to slash the Fremont PD budget and instead use those resources towards solving homelessness. We demand a budget that supports community wellbeing, rather than empowers the police forces that tear them apart.
+  
+  As the City Council, the budget proposal is in your hands. It is your duty to represent your constituents. I am urging you to completely revise the budget for the 2020-2021 fiscal year, and to fund #CareNotCops.
+
+  We can be a beacon for other cities to follow if only we have the courage to change.
+
+  Sincerely,
+
+  \[YOUR NAME\]
+
```
Comment (teddywilson) [inline _emails/fremont-ca.md:38]: nit) the spaces in between these tags is not necessary.

Also, why are there escape characters? Shouldn't it just be

[YOUR NAME]
[YOUR ADDRESS]
etc.

?

Comment (serifluous) [reply _emails/fremont-ca.md]: fixed!

No reason for the escape characters—copied from another template 😬

**Comment by teddywilson** [comment]
Woah, I didn't realize this was Sarah!!! What's up dude!

**Comment by serifluous** [comment]
@teddywilson hahhah heyyy! 👋

**Pull request merged**